src/pages/quiz/creation/Question.ts

```diff
@@ -14,6 +14,21 @@ export type MultipleChoiceQuestion = {
     feedback: string[];
 };
 
+export type SingleChoiceQuestion = {
+    /** The textual content of the question. */
+    question: string;
+    /** The type of the question, indicating it's a "multiple-choice" type. */
```

Comment:
NIT: I guess the description could be changed or deleted

Reply:
I was wondered if it should be and imo it still may be usefull even in single choice

Reply:
sry its late, corrected